test/data/anki-note-builder-test-results.json

```diff
@@ -4361,5 +4361,51 @@
         "url": "<a href=\"url:\">url:</a>"
       }
     ]
+  },
+  {
+    "name": "Test search resolution - word",
```

Comment:
Are you hand-writing these test cases or are these generated?

Reply:
Only `translator-test-inputs.json` is hand-written, the rest is `npm run test:unit:write`